client/config/deploymentSpecificConfig.js

```diff
@@ -17,3 +17,5 @@
  */
 
 tieConfig.constant('SERVER_URL', null);
+
+tieConfig.constant('PRIMER_URL_DIRECTORY', '/docs/');
```

Comment:
PRIMER_DIRECTORY_URL, perhaps? (Since it's a URL to the Primer directory.)

Reply:
yeah I prefer that